### ISSUE TITLE
 A link on a data publisher's page to its corresponding GRSciColl institution page

### ISSUE THREAD

**Issue by spalp** [gbif/registry#564]
For institutions, for which the data publisher and the GRSciColl institution entry are **the same** - either indicated by the fact they share the same ROR ID, or by the fact that the publisher page as [this one](https://www.gbif.org/publisher/826e5964-1925-48e4-a839-f4b418e75260) was added a master source of the GRSciColl entry, e.g. [here](https://registry.gbif.org/institution/3e2c3303-f852-4ea4-9699-2134157bf6fc/master-source), it might be useful to provide a link on the data publisher's page to the GRSciColl institution page...  

In other words, could these data publishers have an additional tab named **GRSciColl** (_as the last tab drawn in the image below_)?
![image](https://github.com/gbif/portal-feedback/assets/11872373/610591b8-caf3-43c3-a3ae-92dbff6b8508)

**Comment by marcos-lg** [comment]
Deployed to production.
